### PR TITLE
Lw identity check

### PR DESCRIPTION
**Pull request by lindawoo-plaid**
1) Add isAuth and isIdentity booleans to make sure products array is correct when creating link token
2) reconfigure the way data is received in routes and passed to create new account in database

### CODE REVIEW

File: client/src/components/UserCard.tsx [hunk 1]
```diff
@@ -28,11 +29,13 @@ export default function UserCard(props: Props) {
   const status = props.item != null ? props.item.status : 'good';
   const isSandbox = PLAID_ENV === 'sandbox';
   const isGoodState = status === 'good';
+  const isAuth = IS_PROCESSOR === 'true' ? false : true;
+  const isIdentity = props.user.should_verify_identity ? true : false;
```
Comment (mattnguyen1) [inline client/src/components/UserCard.tsx:33]: you can probably just remove the ternary operator here and just have `const isIdentity = props.user.should_verify_identity`

File: server/routes/linkTokens.js [hunk 2]
```diff
@@ -24,9 +24,12 @@ router.post(
   '/',
   asyncWrapper(async (req, res) => {
     try {
-      const { userId, itemId } = req.body;
+      const { userId, itemId, isAuth, isIdentity } = req.body;
```
Comment (mattnguyen1) [inline server/routes/linkTokens.js:27]: looks like `isAuth` isn't getting used anywhere; is it supposed to do the similar conditional push to the products array like `identity`?

**Review by mattnguyen1 — APPROVED** [review]
looks good, just left a couple minor comments!

File: server/routes/linkTokens.js (hunk 2 above)
Comment (lindawoo-plaid) [reply server/routes/linkTokens.js:27]: oh, yeah, it is not being used here in the linkTokens route...we use "auth" regardless of whether we are using a processor or not.

**Pull request merged**